scripts/mh_sbuserinfo.pl

```diff
@@ -0,0 +1,344 @@
+				}
+
+				$format = $format . $users_op . '%n:';
+
+				if (Irssi::settings_get_bool('mh_sbuserinfo_show_details_halfop'))
+				{
+					# add halfops to ops so users calculation below matches
+					$users_op = $users_op + $users_ho;
+
+					if ($showmode)
+					{
+						$format = $format . '%%';
+					}
+
+ 					$format = $format . $users_ho . ':';
+				}
+
+				if ($showmode)
+				{
+					$format = $format . '+';
+				}
+
+				$format = $format . $users_vo . ':' . ($users - ($users_op + $users_vo)) . ')';
+			}
+
+			my $limit = $channel->{'limit'};
+
+			if ($limit)
+			{
+				$format = $format . '/';
+
+				if (Irssi::settings_get_bool('mh_sbuserinfo_show_warning_limit'))
+				{
+					my $setting_percent = Irssi::settings_get_int('mh_sbuserinfo_show_warning_limit_percent');
+
+					if ($setting_percent > 100)
+					{
+						$setting_percent = 100;
+					}
+
+					if ($setting_percent < 0)
+					{
+						$setting_percent = 0;
+					}
+
+					my $percent = int(($users / $limit) * 100);
+
+					if ($percent >= $setting_percent)
+					{
+						$format = $format . $warning_format;
+					}
+				}
+
+				$format = $format . $limit . '%n';
+			}
+
+		} else {
+
+			$format = '?'
+		}
+	}
+
+	if ($format ne '')
+	{
+		$statusbaritem->default_handler($get_size_only, '{sb ' . $format . '}', '', 0);
+
+	} else {
+
+		$statusbaritem->default_handler($get_size_only, '{sb }', '', 0);
+	}
+}
+
+##############################################################################
+#
+# script on load
+#
+##############################################################################
+
+sub script_on_load
+{
+	Irssi::settings_add_bool('mh_sbuserinfo', 'mh_sbuserinfo_show_details', 1);
+	Irssi::settings_add_bool('mh_sbuserinfo', 'mh_sbuserinfo_show_details_mode', 0);
+	Irssi::settings_add_bool('mh_sbuserinfo', 'mh_sbuserinfo_show_details_halfop', 0);
+	Irssi::settings_add_bool('mh_sbuserinfo', 'mh_sbuserinfo_show_warning_opless', 1);
+	Irssi::settings_add_bool('mh_sbuserinfo', 'mh_sbuserinfo_show_warning_limit', 1);
+	Irssi::settings_add_int( 'mh_sbuserinfo', 'mh_sbuserinfo_show_warning_limit_percent', 90);
+	Irssi::settings_add_str( 'mh_sbuserinfo', 'mh_sbuserinfo_warning_format', '%Y');
+
+	Irssi::statusbar_item_register('mh_sbuserinfo', undef, 'statusbar_userinfo');
+
+	Irssi::signal_add_last('channel sync',         'signal_channel_sync_last');
+	Irssi::signal_add_last('channel mode changed', 'signal_channel_mode_changed_last');
+	Irssi::signal_add_last('nick mode changed',    'signal_nick_mode_changed_last');
+	Irssi::signal_add_last('nicklist new',         'signal_nicklist_new_last');
+	Irssi::signal_add_last('nicklist remove',      'signal_nicklist_remove_last');
+	Irssi::signal_add_last('setup changed',        'signal_setup_changed_last');
+	Irssi::signal_add_last('window changed',       'signal_window_changed_last');
+}
+
+#
+# start script in a timeout to avoid printing before irssis "loaded script"
+#
+Irssi::timeout_add_once(10, 'script_on_load', undef);
```

Comment:
This is unnecessary, you aren't printing anything and there is no harm in printing before "loaded script" unless in special cases